### PR TITLE
Patch: Interests limit edge case

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- handle edge case for existing profiles with more than 10 interests

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

**Review by didd — APPROVED** [review]

File: extensions/widgets/trending/src/components/cards/latest-topics.tsx [hunk 1]
```diff
@@ -247,7 +247,7 @@ export const LatestTopics: React.FC<LatestTopicsProps> = props => {
     }
 
     // if subscribing, and limit is already reached, return snackbar
-    if (subscribing && localSubscribedTagsRef.current.length === 10) {
+    if (subscribing && localSubscribedTagsRef.current.length >= 10) {
```
Comment (kenshyx) [inline extensions/widgets/trending/src/components/cards/latest-topics.tsx:250]: Unrelated to the changes in this PR, I've took a look at the overall business logic for the topic subscription, it looks very bad, unreadable and overly complicated

**Review by kenshyx — APPROVED** [review]
